README.md

```diff
@@ -8,6 +8,8 @@ It implements the well-known encoder-decoder architecture with attention.
 
 If you are interested in collaborating or have any questions, please submit a pull request or issue.
 You can also send questions to *sockeye-dev-at-amazon-dot-com*.
+
+If you are AWS DeepLearning AMI user please refer to the corresponding section below.
```

Comment:
I think we can leave this sentence out and only mention AWS DLAMI under Installation.